doc/topics/windows/windows-package-manager.rst

```diff
@@ -995,51 +975,46 @@ Firefox.
 installer (str)
 ---------------
 
-This is the path to the binary (``.exe``, ``.msi``) that will install the
-package. This can be a local path or a URL. If it is a URL or a Salt path
-(``salt://``), the package will be cached locally and then executed. If it is a
-path to a file on disk or a file share, it will be executed directly.
+The path to the binary (``.exe``, ``.msi``) to install a  package.
+This can be a local path or a URL. If it is a URL or a Salt path (``salt://``),
+then  the package is cached locally and then executed.
+If it is a path to a file on disk or a file share, then it is executed directly.
 
 .. note::
-    When storing software in the same location as the winrepo it is usually best
-    practice to place each installer in its own directory rather than in the
-    root of winrepo.
+    When storing software in the same location as the winrepo then
```

Comment:
**Nitpick (non-blocking):** Instead of "then" I would just use a colon (:).

Reply:
Updated